engine/hunspell_table.py

```diff
@@ -6284,15 +6293,18 @@ def _command_cancel(self) -> bool:
         '''
         if (self.is_empty()
             and not self._typed_compose_sequence):
-            if self.get_lookup_table().get_number_of_candidates():
+            if (self.get_lookup_table().get_number_of_candidates()
```

Comment:
I tried to test this manually to make sure that switching from `self._update_ui_empty_input()` to `self._clear_input_and_update_ui()` didn't break it. But I couldn't figure out how to get candidates with an empty pre-edit. I tried setting the min chars to 0, but every time the pre-edit was empty, I had no candidates. Do I need to change something else? Do you think switching to `self._clear_input_and_update_ui()` is fine for this case?